field_images/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+project(field_images)
+
+include(CompileWarnings)
+include(GenResources)
+
+if (WITH_JAVA)
+    find_package(Java REQUIRED)
+    include(UseJava)
+
+    file(GLOB_RECURSE JAVA_SOURCES src/main/java/*.java)
+    file(GLOB_RECURSE JAVA_RESOURCES src/main/native/resources/*.json src/main/native/resources/*.png src/main/native/resources/*.jpg)
+    add_jar(field_images_jar SOURCES ${JAVA_SOURCES} RESOURCES NAMESPACE "src/main/native/resources/edu/wpi/first/fields" ${JAVA_RESOURCES} OUTPUT_NAME field_images2)
```

Comment:
```suggestion
    add_jar(field_images_jar SOURCES ${JAVA_SOURCES} RESOURCES NAMESPACE "edu/wpi/first/fields" ${JAVA_RESOURCES} OUTPUT_NAME field_images)
```

Reply:
This fixes both the .jar name and the location of the resources within the jar to match the Gradle build.